log/standardLogger.go

```diff
@@ -14,40 +13,25 @@ import (
 
 const keyFields = "_fields"
 
-type logrusLevelConfig interface {
-	getLogrusLevel() logrus.Level
-}
-
-type ioOutConfig interface {
-	getIoOut() io.Writer
-}
-
 type standardLogger struct {
 	internal *logrus.Logger
 	fields   frozen.Map
 }
 
 func (sf standardFormat) Format(entry *logrus.Entry) ([]byte, error) {
-	message := strings.Builder{}
-	message.WriteString(entry.Time.Format(time.RFC3339Nano))
-	message.WriteByte(' ')
+	sections := append(make([]string, 0, 5), entry.Time.Format(time.RFC3339Nano), strings.ToUpper(entry.Level.String()))
```

Comment:
Interesting idea, does this provide a performance improvement?

Reply:
it's really just because I was frustrated with managing the whitespaces though I remembered I tried comparing `Join` and using the `Builder` (my test was small) and there wasn't really any difference, at least with small cases.